src/groups/bmq/bmqa/bmqa_message.h

```diff
@@ -117,6 +121,8 @@ struct MessageImpl {
     /// SubscriptionHandle this message is associated with
     bmqt::SubscriptionHandle d_subscriptionHandle;
 
+    bsl::shared_ptr<const bmqp::MessageProperties_Schema> d_schema_sp;
```

Comment:
This looks like a private component of `MessageProperties`, we should probably be referring to it through the typedef
```suggestion
    bmqp::MessageProperties::SchemaPtr d_schema_sp;
```